common/check_expired_locked_rules

```diff
@@ -25,28 +36,50 @@ def main():
     '''
     status = OK
     session = get_session()
+
+    base_statement = select(
+        models.ReplicationRule.id,
+        models.ReplicationRule.name,
+        models.ReplicationRule.scope,
+        models.ReplicationRule.rse_expression,
+    )
+
     try:
-        query = "select rawtohex(id), scope, name, rse_expression from atlas_rucio.rules where locked=1 and expires_at<sys_extract_utc(localtimestamp)"
-        print 'Locked expired rules'
-        for row in session.execute(query):
+        expired_locked_rules = base_statement.where(
+            and_(
+                models.ReplicationRule.locked == true(),
+                models.ReplicationRule.expires_at < datetime.utcnow()  # see https://github.com/rucio/rucio/issues/6476
+            )
+        )
+        for row in session.execute(expired_locked_rules):
             status = CRITICAL
-            print row[0], row[1], row[2]
+            print(row[0], row[1], row[2])
+
```

Comment:
This blank line (and the similar one you introduced later) produces a curious grouping. According to PEP 8:

> Use blank lines in functions, sparingly, to indicate logical sections.

A `try` and its associated `except` statements are typically part of the same section. It would make more sense to introduce a blank line before the second `try` statement and before the last `sys.exit(status)` call.